charts/mirrorbits-parent/README.md

```diff
@@ -0,0 +1,102 @@
+# Mirrorbits Parent (All-in-one Mirrorbits shaped for the Jenkins project)
+
+This chart allows to deploys up to three services:
+
+* [mirrorbits](https://github.com/etix/mirrorbits) through the sub-chart <https://github.com/jenkins-infra/helm-charts/tree/main/charts/mirrorbits-lite>
+* [httpd (Apache2)](https://httpd.apache.org/) through the sub-chart <https://github.com/jenkins-infra/helm-charts/tree/main/charts/httpd>
+* [rsyncd](https://linux.die.net/man/1/rsync) through the sub-chart <https://github.com/jenkins-infra/helm-charts/tree/main/charts/rsyncd>
+
+The goal is to deploy either an "HTTP redirector" service centered on [mirrorbits](https://github.com/etix/mirrorbits) and/or a Jenkins download mirror.
+
+## Requirements
+
+This chart has the same requirements as any of the 3 sub-charts.
+
+## Settings (Values)
+
+Look at the [`values.yaml` source file](./values.yaml) to get the possible configuration values.
+
+### Ingress
+
+If you need to provide an ingress to route request to a mix of the sub services, then you can specify the ingress configuration through this parent chart (instead of the sub-charts).
+
+Example to define an Nginx ingress on the domain `downloads.company.org` with TLS enabled (managed by Let's Encrypt),
+where all URLs ending with a `zip` or `gz` string are redirected to the mirrorbits backend,
+and the other requests to httpd:
+
+```yaml
+ingress:
+  enabled: true
+  annotations:
+    "cert-manager.io/cluster-issuer": "letsencrypt-prod"
+    "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+    "nginx.ingress.kubernetes.io/use-regex": "true"
+  hosts:
+    - host: downloads.company.org
+      paths:
+        - path: /
+          backendService: httpd
+        - path: /.*[.](zip|gz)$
+          backendService: mirrorbits
+  tls:
+    - secretName: ingress-certificates
+      hosts:
+        - downloads.company.org
+
+mirrorbits-lite:
+  enabled: true
+
+httpd:
+  enabled: true
+```
+
+### Storage
+
+This chart is usually deployed with the same persistent storage for all the sub-services.
+As such you can manage a common persistent volume (and associated claims) once and for all instead of delegating to the sub-charts.
+
+Since the chart is initially designed to run on AKS with an Azurefile volume, part of the PV/PVC configuration can be managed (CSI setup, Azurefile secret for the access key)
+through the `storage.persistentVolume.azureFile` value.
+
+Example with an Azure file PV and associated PVC mounted in read only:
+
+```yaml
+mirrorbits-lite:
+  enabled: true
+  repository:
+    name: <name of the PVC>
+    reuseExistingPersistentVolumeClaim: true
+
+httpd:
+  enabled: true
+  repository:
+    name: <name of the PVC>
```

Comment:
```suggestion
    name: myExistingPersistentVolumeClaimName
    reuseExistingPersistentVolumeClaim: true

httpd:
  enabled: true
  repository:
    name: myExistingPersistentVolumeClaimName
```

Reply:
I would want to keep it that way because, until we have a solution to pass the PVC generated name to the subcharts, the end users (e.g. us) have to determine the PVC name and set it statically in the values.

I know it is annoying, but I don't want the documentation to be more confusing around "is it the umbrella PVC created  the parent chart or another PVC?".

Is that ok for you to keep it that way until then?